### PR TITLE
Add CI/CD In The NextJS App And Test It Part 4

### PR DESCRIPTION
**Pull request by qur786**
**Work done:**

- Added `ci.yml` workflow

**Note:**

- When this PR will be merged testing can be done, by pushing commits to the main branch.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+name: Componly
+
+on:
+  push:
+    branches:
+      - main
```
Comment (nik-1207) [inline .github/workflows/ci.yml:6]: @qur786  push a commit replacing this name with current branch name
and the revert it. 
updating the name here will trigger ci and we will be able to see the working

Comment (qur786) [reply .github/workflows/ci.yml:6]: good idea.

**Comment by qur786** [comment]
@maneetgoyal sir, can you please add the required secret tokens?

**Comment by maneetgoyal** [comment]
> @maneetgoyal sir, can you please add the required secret tokens?

Done.

**Comment by qur786** [comment]
> > @maneetgoyal sir, can you please add the required secret tokens?
> 
> Done.

@maneetgoyal sir, 
Still unauthorized error is coming up.
image

**Comment by qur786** [comment]
CI/CD passing.

**Pull request merged**